Tools/autotest/sim_vehicle.py

```diff
@@ -948,6 +977,9 @@ def generate_frame_help():
     if not cmd_opts.no_rebuild:  # i.e. we should rebuild
         do_build(vehicle_dir, cmd_opts, frame_infos)
 
+    if cmd_opts.fresh_params:
+        do_build_parameters(vehicle_dir)
```

Comment:
Why not pass `cmd_opts.vehicle` instead of the directory? The method wouldn't need to get the basename then.

Reply:
Fixed.  I just didn't bother looking up the option name :-)